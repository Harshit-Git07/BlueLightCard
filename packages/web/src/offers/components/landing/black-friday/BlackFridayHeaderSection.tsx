import Image from 'next/image';
import Link from '@/components/Link/Link';

const BlackFridayHeaderSection = () => (
  <div className="bg-black">
    <Link href="/index.php" useLegacyRouting>
      <Image
        className="mx-auto"
        src="https://cdn.bluelightcard.co.uk/web/logos/blc-logo-black-friday-2023.png"
        alt="logo"
        width={323}
        height={66}
      />
    </Link>
  </div>
);

export default BlackFridayHeaderSection;
