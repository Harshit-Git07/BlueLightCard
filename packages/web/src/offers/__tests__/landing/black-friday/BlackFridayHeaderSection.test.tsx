import renderer from 'react-test-renderer';

import BlackFridayHeaderSection from '@/offers/components/landing/black-friday/BlackFridayHeaderSection';

describe('BlackFridayHeaderSection', () => {
  it('should match snapshot', () => {
    const component = renderer.create(<BlackFridayHeaderSection />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
