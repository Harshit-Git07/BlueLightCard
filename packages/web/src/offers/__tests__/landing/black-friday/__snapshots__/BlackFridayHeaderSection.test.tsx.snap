// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlackFridayHeaderSection should match snapshot 1`] = `
<div
  className="bg-black"
>
  <a
    className="dark:text-palette-secondary"
    data-testid="anchor-link"
    href="/index.php"
  >
    <img
      alt="logo"
      className="mx-auto"
      data-nimg="1"
      decoding="async"
      height={66}
      loading="lazy"
      onError={[Function]}
      onLoad={[Function]}
      src="/_next/image?url=https%3A%2F%2Fcdn.bluelightcard.co.uk%2Fweb%2Flogos%2Fblc-logo-black-friday-2023.png&w=750&q=75"
      srcSet="/_next/image?url=https%3A%2F%2Fcdn.bluelightcard.co.uk%2Fweb%2Flogos%2Fblc-logo-black-friday-2023.png&w=384&q=75 1x, /_next/image?url=https%3A%2F%2Fcdn.bluelightcard.co.uk%2Fweb%2Flogos%2Fblc-logo-black-friday-2023.png&w=750&q=75 2x"
      style={
        {
          "color": "transparent",
        }
      }
      width={323}
    />
  </a>
</div>
`;
